File: Code/Account.cpp
#include <iostream>
#include <string>

using namespace std;

bool login() {
    string username, password;

    cout << "Inserisci il tuo nome utente: ";
    cin >> username;

    cout << "Inserisci la tua password: ";
    cin >> password;

    if (username == "Administrator" && password == "segreto") {
        cout << "Accesso consentito." << endl;
        return true;
    } else {
        cout << "Nome utente o password errati. Accesso negato." << endl;
        return false;
    }
}

int main() {
    login();

    return 0;
}

File: Account.py
def login():
    username = input("Inserisci il tuo nome utente: ")
    password = input("Inserisci la tua password: ")

    if username == "admin" and password == "segreto":
        print("Accesso consentito.")
    else:
        print("Nome utente o password errati. Accesso negato.")

login()